docs/src/rules/no-underscore-dangle.md

```diff
@@ -28,7 +28,6 @@ Examples of **incorrect** code for this rule:
 var foo_;
 var __proto__ = {};
 foo._bar();
-const [_foo, ..._bar] = list;
 ```
```

Comment:
It's present in markdown but I don't see it in [the latest deployed docs](https://eslint.org/docs/latest/rules/no-underscore-dangle#rule-details), 

<img width="809" alt="Screenshot 2022-12-24 at 3 57 27 PM" src="https://user-images.githubusercontent.com/46647141/209431938-65662ffd-0606-4fd8-a6f7-cdb661a76426.png">

Reply:
It was added in https://github.com/eslint/eslint/pull/16006 which was merged after the release. It's visible in the head version:

https://eslint.org/docs/head/rules/no-underscore-dangle